corehq/apps/fixtures/dispatcher.py

```diff
@@ -19,4 +19,5 @@ class FixtureInterfaceDispatcher(ProjectReportDispatcher):
 
     @method_decorator(require_can_edit_fixtures)
     def dispatch(self, request, *args, **kwargs):
+        request.use_requirejs = True
```

Comment:
This is the major on/off switch. I'd rather set a flag inside the templates themselves, rather than on the request object, because I think the way migration would work is by converting one base template at a time...but I don't think it's possible to set a flag in `fixtures_base.html` and have that affect `base.html`.